src/components/TableHeadCell.js

```diff
@@ -33,7 +33,8 @@ const defaultHeadCellStyles = theme => ({
     cursor: 'pointer',
     paddingLeft: '4px',
     height: '10px',
-  },
+    position: 'absolute'
+},
```

Comment:
Needs a tab here to keep the bracket in line with the others.

Reply:
Sounds good - thanks!